src/app/containers/TopStories/index.test.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+
+import { ServiceContextProvider } from '#contexts/ServiceContext';
+import { RequestContextProvider } from '#contexts/RequestContext';
+
+import TopStories from '.';
+import topStories from '#pages/StoryPage/topStories.json';
+
+// eslint-disable-next-line react/prop-types
+const renderTopStories = ({
+  content = topStories,
+  bbcOrigin = 'https://www.test.bbc.co.uk',
+} = {}) => {
+  return render(
+    <ServiceContextProvider service="pidgin">
+      <RequestContextProvider
+        bbcOrigin={bbcOrigin}
+        isAmp={false}
+        pageType="STY"
+        pathname="/pidgin/tori-49450859"
+        service="pidgin"
+        statusCode={200}
+      >
+        <TopStories content={content} enableGridWrapper />
```

Comment:
I can't see any reference to `enableGridWrapper` on this PR, has it been removed?

Reply:
Thanks. It's been removed.